README.md

```diff
@@ -297,6 +297,8 @@ hazelcast-client:
 
 **Note:** Hazelcast Client outside Kubernetes cluster works only in the **Kubernetes API** mode (it does not work in the **DNS Lookup** mode).
 
+**Note:** To run external smart client your Kubernetes cluster must have an external public IP. Minikube, for example, won't work.
```

Comment:
What about using minikube tunnel? I always use it in replacement of Loadbalancer.
https://minikube.sigs.k8s.io/docs/handbook/accessing/

Reply:
I didn't know about `minikube tunnel`. I'll check if it works.